assets/android/kotlin/CpdDark.kt

```diff
@@ -162,6 +162,35 @@ object DarkDesignTokens {
   val cpdColorGray300 = Color(0xffe2e6ea)
   val cpdColorGray200 = Color(0xffeef1f3)
   val cpdColorGray100 = Color(0xfff7f8f9)
+  val cpdFontLetterSpacingHeadingXl = 0
```

Comment:
The output, once again looks wrong. I'll do the same exercise with the Android team